README-minishift.md

```diff
@@ -202,43 +208,50 @@ The above created:
 Now deploy the CRDs that determine which Kubernetes resources are federated across
 the clusters:
 
-    oc project federation-system
-    oc create -n federation-system -f federatedtypes/
+~~~sh
+oc project federation-system
+oc create -n federation-system -f federatedtypes/
+~~~
 
 After a short while the federation controller manager pod is running:
 
-    oc get pod -n federation-system
+~~~sh
+oc get pod -n federation-system
 
-    NAME                              READY     STATUS    RESTARTS   AGE
-    federation-controller-manager-0   1/1       Running   0          31s
-
-
-<a id="orgbae3b28"></a>
+NAME                              READY     STATUS    RESTARTS   AGE
+federation-controller-manager-0   1/1       Running   0          31s
+~~~
 
+<a id="markdown-register-the-clusters-in-the-cluster-registry" name="register-the-clusters-in-the-cluster-registry"></a>
 ## Register the clusters in the cluster registry
 
 Verify that there are no clusters in the registry yet (but note
 that you can already reference the CRDs for federated clusters):
 
-    oc get federatedclusters -n federation-system
-    oc get clusters --all-namespaces
+~~~sh
+oc get federatedclusters -n federation-system
+oc get clusters --all-namespaces
 
-    No resources found.
+No resources found.
+~~~
 
 Now use the `kubefed2` tool to register (*join*) the two clusters:
 
-    kubefed2 join cluster1 \
-             --host-cluster-context cluster1 \
-             --add-to-registry \
-             --v=2
-    kubefed2 join cluster2 \
-             --host-cluster-context cluster1 \
-             --add-to-registry \
-             --v=2
+~~~sh
+kubefed2 join cluster1 \
+            --host-cluster-context cluster1 \
+            --add-to-registry \
+            --v=2
+kubefed2 join cluster2 \
+            --host-cluster-context cluster1 \
+            --add-to-registry \
+            --v=2
+~~~
 
 Verify that the federated clusters are registered and in a ready state (this
 can take a moment):
 
+~~~yaml
```

Comment:
Same comment as above: not sure this should be marked as YAML

Reply:
It's mostly shell, but as output it's yaml format it better colourizes the outcome

Reply:
```suggestion
~~~sh
```